types/express-list-endpoints/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for express-list-endpoints 4.0
+// Project: express-list-endpoints
```

Comment:
I think you can fix the last lint issue by changing this to `Project: https://github.com/AlbertoFdzM/express-list-endpoints`

Reply:
I'll give it a try.

Reply:
👍

Reply:
@amcasey looks like it worked! Thanks for the suggestion.